static/js/common/upload-addon.js

```diff
@@ -27,7 +27,12 @@
     }
 
     $.fn.addonUploader = function( options ) {
-        var settings = {'filetypes': ['zip', 'xpi', 'crx', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
+        var settings = {
+            'filetypes': ['zip', 'xpi', 'crx', 'jar', 'xml'],
+            'getErrors': getErrors,
+            'cancel': $(),
```

Comment:
Well, that's the lazy variant of someone requiring a jquery-element (e.g `$('a.cancel')`) who don't want to bother checking if the option is given or not. `upload-addon.js` also re-implements some of the cancelation logic but right now I didn't want to fix this, I'd rather have a follow-up that refactors that code with proper tests…